runtime/shiden/Cargo.toml

```diff
@@ -112,6 +112,9 @@ moonbeam-evm-tracer = { workspace = true, optional = true }
 moonbeam-rpc-primitives-debug = { workspace = true, optional = true }
 moonbeam-rpc-primitives-txpool = { workspace = true, optional = true }
 
+# TODO: see if this can be solved in a better way - right now I don't see how else to inject `runtime-benchmarks` into the `polkadot-runtime`
+polkadot-runtime = { workspace = true, optional = true }
```

Comment:
Runtime cannot depend on other runtime  during wasm build, it will bring troubles with version constant. I see here it’s optional and probably will not used in production builds, but let’s be careful anyway.

Reply:
This doesn't really a new dependency - it only allows me to enable `runtime-benchmarks` features for an already existing dep. Otherwise, `pallet-xcm` used by `polkadot-runtime` will break during compilation.

I did a bit of digging through `cargo tree` output, and this is what essentially brings `polkadot-runtime` dep into the compilation:
├── **cumulus-pallet-parachain-system v0.1.0**
│   ├── **cumulus-primitives-parachain-inherent v0.1.0** 
... and deep down this subtree, we have polkadot client and runtime deps

And unfortunately, enabling **runtime-benchmarks** for `cumulus-pallet-parachain-system` doesn't help since it doesn't propagate the feature enabling down to `polkadot-runtime`.